docs/scripting/language/reference/04-Functions.md

```diff
@@ -1348,4 +1348,27 @@ factorial(3) \\ factorial initiate
                if(0 == 0) return 1 \\ checks the conition which is true and return 1
 		\\ at the final call 3 * 2 * 1 * 1 
 ```
+### Stack Memory
+The stack is a region of memory used for storing local variables, function call information, and control flow data. It operates in a Last-In-First-Out (LIFO) manner, which means that the last item pushed onto the stack is the first one to be popped off.
+#### Example (Stack Overflow)
+```c
+#pragma dynamic 35 // (35 * 4 bytes, a cell size) #pragma dynamic [cells] helps to modify the size of stack, read docs/scripting/language/Directives to know more about #pragma
+main(){
+	grow_stack(1);
+}
+grow_stacK(n){ // recursive function
+	printf("N: %d", n);
+	grow_stacK(n+1);
+}
+```
+#### Output
+```
+N: 1
+N: 2
+N: 3
+.. .
+Stack/heap collision (insufficient stack size)
+```
+![Stack](https://cdn.discordapp.com/attachments/966187760341901352/1138405264626815017/stack.png)
```

Comment:
It'll be worth hosting this image on the actual open.mp site or on an image host such as imgur - Discord's URL structure could change or the CDN could go down and it's not a dedicated image host so I wouldn't want to depend on it.